iceoryx_utils/platform/mac/include/iceoryx_utils/platform/grp.hpp

```diff
@@ -15,3 +15,17 @@
 #pragma once
 
 #include <grp.h>
+#include <unistd.h>
+
+// linux version looks like
```

Comment:
should be doxygen here

Reply:
Actually no. This is a developer comment to clarify the implementation and not intended for the documentation.
For instance, this comment should help you during this code review to understand the intention behind creating iox_sem_close and not using it directly.